client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -18,11 +111,80 @@ export default function PaymentMethodAdd(): ReactElement {
 				<CardHeading size={ 36 }>{ translate( 'Payment Method' ) }</CardHeading>
 			</div>
 
-			<Card className="payment-method-add__body">
-				<div className="payment-method-add__body-left"></div>
+			<CheckoutProvider
+				onPaymentComplete={ () =>
+					onPaymentSelectComplete( {
+						successCallback: () => page( '/partner-portal/payment-method/' ),
+						translate,
+						showSuccessMessage,
+					} )
+				}
+				showErrorMessage={ showErrorMessage }
+				showInfoMessage={ showInfoMessage }
+				showSuccessMessage={ showSuccessMessage }
+				paymentMethods={ paymentMethods }
+				paymentProcessors={ {
+					card: ( data ) =>
+						assignNewCardProcessor(
+							{
+								useAsPrimaryPaymentMethod,
+								translate,
+								stripe,
+								stripeConfiguration,
+								dispatch,
+							},
+							data as NewCardSubmitData
+						),
+				} }
+				isLoading={ isStripeLoading }
+				initiallySelectedPaymentMethodId="card"
+			>
+				<Card className="payment-method-add__content">
+					<CardHeading>{ translate( 'Credit card details' ) }</CardHeading>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Stored Credit Card Details' )` *ES Score: 9*